mlflow/store/artifact/s3_artifact_repo.py

```diff
@@ -76,7 +77,7 @@ def parse_s3_uri(uri):
     def get_s3_file_upload_extra_args():
         import json
 
-        s3_file_upload_extra_args = os.environ.get("MLFLOW_S3_UPLOAD_EXTRA_ARGS")
+        s3_file_upload_extra_args = MLFLOW_S3_UPLOAD_EXTRA_ARGS.get()
```

Comment:
Looks like we might be missing a few other environment variables: https://github.com/mlflow/mlflow/blob/6c1882d5b4f50bdb0c3694a4d12dddc5e16af4ff/mlflow/store/artifact/s3_artifact_repo.py#L87-L101